doc/client-howto.md

```diff
@@ -141,18 +159,22 @@ but will not be able to retrieve any files.
 
 * `KACHERY_STORAGE_DIR`: If set, this defines the directory in
 which the client will look for files locally before attempting
-to obtain them from the node. QUERY: The environment variable
-will be honored over the config file.
+to obtain them from the node. It must coincide with the $KACHERY_STORAGE_DIR
+for the running daemon. If the daemon is running on a different
+computer it is okay if the literal value of $KACHERY_STORAGE_DIR
+is different as long as it is referring to the same underlying
+directory.
 * `KACHERY_DAEMON_PORT`: If set, this variable will define an
 alternate port to use when connecting to the kachery daemon (node).
 * `KACHERY_DAEMON_HOST`: If set, this variable will define the
 host on which to look for the node. If not set, the daemon
 is assumed to exist on localhost (i.e. the node is assumed
-to be running on the same machine as the client). **CONFIRM**
+to be running on the same machine as the client).
 * `KACHERY_TEMP_DIR`: If set, temporary files will be stored in
-this directory.
+this directory. Otherwise, a default location will be used.
```

Comment:
Might as well state what the default is, unless it's likely to change.

Reply:
Okay, will do.